pom.xml

```
@@ -13,21 +13,23 @@
     <groupId>org.mule.connectors</groupId>
     <artifactId>mule-sockets-connector</artifactId>
     <packaging>mule-extension</packaging>
-    <version>1.3.0-SNAPSHOT</version>
+    <version>1.3.1-SNAPSHOT</version>
```

We should have the next major version, do you mind adding suggestion to this PR?
```suggestion
    <version>2.0.0-SNAPSHOT</version>
```